renovate.json

```diff
@@ -2,5 +2,20 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "matchPackagePrefixes": ["ch.qos.logback:"],
+      "groupName": "logback packages"
```

Comment:
this should group logback dependency updates into a single PR (many other java dependencies already have [built-in grouping by renovate](https://docs.renovatebot.com/presets-group/), but not logback)